app.rb

```diff
@@ -1,20 +1,63 @@
+require "pry"
+# require_relative "models/apartment"
+# require_relative "models/tenant"
 require_relative "data"
 apartments = data[:apartments]
 tenants = data[:tenants]
 
+
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
   # Explain how the data is structured
+    # The data is structured in tow array apartments and tenants.
+    # Each of these array contains multiple objects to show detil informtion about apartments and tenants.
   # What are the properties for each of the two types of hashes
+    # apartments and tenants
+
 
 # Use enumerables to -
   # Print all the addresses for the apartments
+  apartments.map do |i|
+    puts "#{apartments.[i][:address]}"
+  end
   # Print all the names for tenants
+  tenants.map do |i|
+    puts "#{tenants.[i][:name]}"
+  end
+
   # Print only apartments that are less then 700 in rent
+cheapApartments= {}
+  apartments.map do |i|
```

Comment:
`.map` returns an array, so you shouldn't have to push into an outside array here.